data/hai/hai reveal 5 philosophers.txt

```diff
@@ -1276,84 +1276,129 @@ mission "Hai Reveal [B13] Freedom"
+			`	The squads move room to room, neutralizing all opposition with ease, until their opponents switch tactics. The pirates begin using electromagnetic pulse devices to shut down the Syndicate suits (and all other electronics nearby), leaving affected operatives helpless for ten long seconds. Here, some of General Choot'k's Unfettered show their strength, outmaneuvering the pirates long enough for the suits to recharge. They dart between operatives and bound off walls with pulse rifles and swords. They strike rapidly and in an organized manner, before withdrawing as the operatives suits re-engage, preventing the pirates from coordinating a counter-assault.`
+			`	After a bloody struggle, forty-three operatives reach the end of the fight, clearing the way to a blast door. Only a couple dozen operatives are known to be seriously wounded, but many are missing in action because the pirates' devices were very effective at trapping the suits. The Unfettered fared comparatively better, as the pirates focused most attacks on the Syndicate mechs, letting the fast, maneuverable Hai pass through. Still, over a dozen of them will take their own tails home as trophies, after the pirates used a miner's cutting power-arm extensions to great effect.`
+			`	The blast door that the force now stands before is twelve meters high and wide, and scanners show it to be six meters deep. It is designed to be closed from this side, though, and has not been sealed well against being reopened. Soon it grinds part-way open, evidently sabotaged, and the operatives put on their goggles and pass through into the darkness of the next room.`
+			`	You could be forgiven for hesitating to follow them, but General Choot'k appears beside you and places one strong paw on your shoulder. "Come, now. We will discover if our task is finished, or impossible. It is time for bravery." He guides you in after the operatives, along with his own troops.`
 			choice
-				`	(Chat with the Elenctic citizens before you go.)`
-				`	(It's been a long day. Leave for your ship.)`
+				`	"Lead on."`
+					goto leadon
+				`	"Are you calling me a coward? If I had more time, I'd challenge you to a duel for that."`
+			`	"Heh heh," he laughs in a low voice. "Maybe soon."`
+			action
+				set "Hai Reveal [B13] Freedom: threatened Choot'k"
```

Comment:
```suggestion
				set "Hai Reveal [B13] Freedom: challenged Choot'k"
```
Even with the original phrasing, this is far more a challenge than a threat, and this situation is one where the difference is quite important.